boto/ec2/connection.py

```diff
@@ -4406,7 +4406,8 @@ def get_all_instance_types(self):
         return self.get_list('DescribeInstanceTypes', params, [('item', InstanceType)], verb='POST')
 
     def copy_image(self, source_region, source_image_id, name=None,
-                   description=None, client_token=None, dry_run=False):
+                   description=None, client_token=None, encrypted=None,
```

Comment:
I think we'd need to add `kms_key_id` and `encrypted` after `dry_run` to preserve backwards compatibility for people using positional args.